HLTrigger/Configuration/python/HLT_75e33_timing_cff.py

```diff
@@ -221,6 +221,24 @@
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/HLTriggerFinalPath_cff")
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/HLTAnalyzerEndpath_cff")
 
+# Load and configure the FastTimerService
+fragment.load("HLTrigger/Configuration/HLT_75e33/services/FastTimerService_cfi")
+fragment.FastTimerService.enableDQM = False
+fragment.FastTimerService.enableDQMbyModule = False
+fragment.FastTimerService.enableDQMbyPath = False
+fragment.FastTimerService.jsonFileName = 'Phase2Timing_resources.json'
+
+# Load and configure the ThroughputService
+fragment.load("HLTrigger/Configuration/HLT_75e33/services/ThroughputService_cfi")
+fragment.ThroughputService.eventRange = 1000
+fragment.ThroughputService.eventResolution = 50
+fragment.ThroughputService.printEventSummary = True
+fragment.ThroughputService.enableDQM = False
+
+# Cusotmize the output as well
+if fragment.HLTriggerFinalPath.contains(fragment.hltTriggerSummaryAOD):
+    fragment.HLTriggerFinalPath.remove(fragment.hltTriggerSummaryAOD)
```

Comment:
I am unhappy about this removal due to ```weird behaviour``` as it typically never gets fixed. This module works fine in the Run-II workflows, and AFAIK HLT Phase 2 now runs like that (?), so why/what is the problem?

Reply:
Ciao @Martin-Grunewald I understand your concerns about weird behaviour getting removed and then lost. 

The issues I see are the following:
1. Adding this module will trigger the EGAMMA/Pixel/MUON and PF reconstruction on all events, i.e., the corresponding modules will run 100% of the times.
2. The timing measurements become extremely difficult to understand. To give you an example, please look at the comparison between CPU-time and real-time here:
* [cpu-time](https://rovere.web.cern.ch/rovere/Phase2_HLT/circles/web/piechart.php?local=false&dataset=13_1_0%2FTTbar200PU_FixUnpackers%40785513d984b_Phase2TimingMenu_8j_8t_4s&resource=time_thread&colours=default&groups=hlt&threshold=0)
* [real-time](https://rovere.web.cern.ch/rovere/Phase2_HLT/circles/web/piechart.php?local=false&dataset=13_1_0%2FTTbar200PU_FixUnpackers%40785513d984b_Phase2TimingMenu_8j_8t_4s&resource=time_real&colours=default&groups=hlt&threshold=0)

While investigating the first point I realized the `triggerSummaryAOD` was introducing a dependency on those modules that, if present, will run 100%.
Removing the `triggerSummaryAOD` solved both issues.

Given that this module is not used, I think removing it while understanding how to solve those issues (and, most importantly, why they show up) is a safe way to proceed. But we can iterate the discussion here.

For the backport to 13_1_X, instead, I would just approve it, since these issues already caused troubles while preparing the results and measurements for the Annual Review, and we have a coming P2UG review in just one month.

Reply:
That's already an issue/problem: the module itself should not trigger the execution of other modules otherwise not run.

Reply:
From a chat with @fwyzard , he suggests to make HLTriggerFinalPath a ```cms.FinalPath``` - could you please try if it solves the issue?

Reply:
I had the same chat with @fwyzard and a simple change from `cms.Path` to `cms.FinalPath` is not welcome by the framework.

```
----- Begin Fatal Exception 03-Oct-2023 15:03:21 CEST-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named step4_HLT.py
Exception Message:
 unknown python problem occurred.
AttributeError: 'str' object has no attribute 'label_'

At:
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_3_X_2023-10-02-2300/src/FWCore/ParameterSet/python/Config.py(1289): <listcomp>
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_3_X_2023-10-02-2300/src/FWCore/ParameterSet/python/Config.py(1289): _insertPaths
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week1/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_3_X_2023-10-02-2300/src/FWCore/ParameterSet/python/Config.py(1488): fillProcessDesc
  <string>(2): <module>

----- End Fatal Exception -------------------------------------------------
```

Reply:
OK, looks like more digging is needed but should not hold up this PR. Thanks for the try!

Reply:
So, turns out we cannot have `EDProducer`s or `EDFilter`s in a `FinalPath`.

@makortel is there a way to run a single `EDProducer` _without_ triggering any unscheduled execution ?

The idea is to "collect" a set of products _if they have already been produced_, but not make their producers run if they did not.

Reply:
> So, turns out we cannot have `EDProducer`s or `EDFilter`s in a `FinalPath`.

Correct, `FinalPath` is intentionally limited to `OutputModule`s.

> is there a way to run a single `EDProducer` _without_ triggering any unscheduled execution ?

No, the only way to control whether an `EDProducer` gets run or not, is to place it in to a `Path` behind an `EDFilter` that controls whether the `EDProducer` is run or not.

Reply:
> No, the only way to control whether an `EDProducer` gets run or not, is to place it in to a `Path` behind an `EDFilter` that controls whether the `EDProducer` is run or not.

Well, this makes the whole unscheduled execution model unusable for the HLT.

Reply:
And, in fact, for anything that uses filters in a meaningful way.